src/modules/form/util/formUtil.ts

```diff
@@ -1476,6 +1476,36 @@ export const parseOccurrencePointFormDefinition = (
 
   return { displayTitle, isEditable, readOnlyDefinition };
 };
+
+export const getLeftNavItems = (
+  formDefinition: FormDefinitionFieldsFragment | undefined | null,
+  itemMap: ItemMap | undefined | null,
+  initialValues: Record<string, any>,
+  localConstants: LocalConstants,
+  minGroupsForLeftNav: number = 3
+) => {
+  if (!formDefinition || !itemMap) return false;
+
+  let leftNavItems = formDefinition.definition.item.filter(
+    (i) => i.type === ItemType.Group && !i.hidden
+  );
+
+  if (leftNavItems.length < minGroupsForLeftNav) return false;
```

Comment:
This check should probably happen after we remove the disabled groups?

Reply:
Yes, looks like the [previous code](https://github.com/greenriver/hmis-frontend/blob/9dcf0821612a3d4e6674e0c3709ca4c3f2326d9b/src/modules/form/components/EditRecord.tsx#L99-L110) did the check in both places but I agree it could be cleaned up! 🧹